fabtests/pytest/efa/test_efa_info.py

```diff
@@ -1,7 +1,16 @@
 import pytest
+from common import UnitTest
+from efa_common import efa_retrieve_gid
 
 @pytest.mark.unit
 def test_efa_info(cmdline_args):
-    from common import UnitTest
     test = UnitTest(cmdline_args, "fi_efa_info_test")
     test.run()
+
+@pytest.mark.unit
+def test_comm_getinfo(cmdline_args):
```

Comment:
So before this bug fix, this test is actually not run for efa? and the bug is not caught?

Reply:
The test was run, but the dest address used was ip address. efa does not recorganize that address, therefore return FI_ENODATA (61). `fi_getinfo_test`t consider FI_ENODATA as a valid return.